concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
Why is this being added here?  I'm not sure I see what it has to do with using RDS.